### ISSUE TITLE
Sum of monthly exports from individual contributor portal don't add up to annual export 

### ISSUE THREAD

**Issue by jvstaak** [fecgov/FEC#10994]
Hi all - 

I am looking to export data for individual contributions made in Virginia for 2019 using the Individual Contributions portal: https://www.fec.gov/data/receipts/individual-contributions

Due to the 500k row limit, I am exporting each month (or a stretch of multiple months depending on file size limits) of 2019 and performing a union to get all contributions made in 2019. 

The screenshot below shows the number of records when filtering for 01/01/2019 - 12/31/2019 (2.38 million) and the smaller exports using individual months or stretches of 2-3 months that covers 01/01/2019 - 12/31/2019. The row sum of the 7 individual exports is only 1.66 million. I'm not sure why the row sum wouldn't be the 2.38 million. Can you advise on what might be going wrong with my query? 

Thanks!

![image](https://user-images.githubusercontent.com/86480669/123437907-c95b9d80-d595-11eb-97f0-278de4f1eec8.png)

**Comment by PaulClark2** [comment]
@jvstaak the first number is an estimate, “about 2,384,000.”  All row counts over 500,000 are estimates.

**Comment by jvstaak** [comment]
@PaulClark2  -- thank you! So the sum of all 12 months queried individually should be equivalent to querying the full 12 months at once, is that correct?

**Comment by PaulClark2** [comment]
@jvstaak that's correct as long as the row count of a single month is not greater than 500,000. If the row count of a single month is greater than 500,000 it would be an estimate.
